.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sd
+    docker:
+      - image: msheiny/pressfreedomci:latest
+        environment:
+          FPF_CI: true
+          CI_SD_ENV: staging
+          CI_AWS_TYPE: t2.small
+          FPF_GRSEC: false
+          TEST_REPORTS: /root/sd
+    steps:
+      # This is really dumb that Im doing this, basically
```

Comment:
ehh so what happened when i tested this...

Basically say you have a working directory defined, like i have `working_directory: ~/sd` .
Securedrop kept getting cloned to `~/sd/securedrop` and then all future commands were running out of `~/sd` . I tried to clone to `~/sd` but obviously git complains that the directory already exists.  Now that i look at the docs.. I could have actually just defined a different `working_directory` for each instance of `run` ..... actually yeah let me do that. its the same number of lines and then i lose all that comment crap.